base/applications/kbswitch/indicdll/indicdll.c

```diff
@@ -195,10 +196,12 @@ DllMain(IN HINSTANCE hinstDLL,
             if (!bAlreadyExists)
             {
                 ZeroMemory(g_pShared, sizeof(*g_pShared));
-                g_pShared->hKbSwitchWnd = FindWindow(INDICATOR_CLASS, NULL);
                 InitializeCriticalSection(&g_pShared->csLock);
                 g_bCriticalSectionInitialized = TRUE;
             }
+
+            if (!g_pShared->hKbSwitchWnd)
+                g_pShared->hKbSwitchWnd = FindWindow(INDICATOR_CLASS, NULL);
```

Comment:
Something else seems to be going on.
If, in the original case, there was a claimed `g_pShared->hKbSwitchWnd` initialization problem, then, it's probably not the only one being problematically initialized....